tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/conversion/MavenConversionTest.java

```diff
@@ -207,8 +207,8 @@ private void createWithRuntime(){
 		new ContextMenuItem("Configure","Convert to Maven Project").select();
 		new DefaultShell("Create new POM");
 		new PushButton("Finish").click();
-		new DefaultShell("Convert to Maven Dependencies");
-		new WaitUntil(new ControlIsEnabled(new PushButton("Finish")), TimePeriod.LONG);
+		new WaitUntil(new ShellIsAvailable("Convert to Maven Dependencies"), TimePeriod.getCustom(2500));
```

Comment:
Wait over 40 minutes? Is it needed to wait for so long? It looks suspicious.

Reply:
Fixed. (I have thought that this could fix some failing unstable tests, but after looking into Jenkins I found out, that 40 minutes is really overkill that fix nothing :-))